docs/_resources/changelog.md

```diff
@@ -61,6 +61,7 @@ Legend:
 > 1. Analytics engine – Ingest cost data into an Azure Data Explorer cluster.
 > 2. Auto-backfill – Backfill historical data from Microsoft Cost Management.
 > 3. Retention – Configure how long you want to keep Cost Management exports and normalized data in storage.
+> 4. Added documentation to use a SAS token to configuring Power BI reports.
```

Comment:
nit: Since you're committing directly to dev, you can put this in the 0.5 section under Power BI > General.

Reply:
Sure I will make the changes. Can we have a quick call to clarify some questions.

Reply:
@Zeo-shark This now needs to go under 0.6 as we have already released 0.5 - Ping me when you are online and I can help you with that.